Cargo.toml

```diff
@@ -43,6 +43,7 @@ rand = { version = "0.8" }
 log = { version = "0.4", optional = true }
 # WASM API
 wasm-bindgen = { version = "0.2.87", optional = true }
+hax-lib = { git = "https://github.com/hacspec/hax/", branch = "karthik/core-for-kyber" }
```

Comment:
This will need to be updated before we can merge this.
Has this all been in https://github.com/hacspec/hax/pull/345?

Reply:
Yes, `karthik/core-for-kyber` and `main` have now the same `proofs-libs`